includes/Admin/Tools.php

```diff
@@ -74,7 +75,7 @@ public function process_data($csv)
         }
         if (count($data) > 0) {
             $oldData = get_option(SIMPLE301REDIRECTS_SETTINGS_NAME);
-            $value = (!empty($oldData) ? array_unique(array_merge(get_option(SIMPLE301REDIRECTS_SETTINGS_NAME), $data)) : $data);
+            $value = (!empty($oldData) ? array_merge(get_option(SIMPLE301REDIRECTS_SETTINGS_NAME), $data) : $data);
```

Comment:
Use it like this in the future, in case settings are empty! though this won't run if settings are empty! (maybe)

```php
get_option(SIMPLE301REDIRECTS_SETTINGS_NAME, [])
```